BootloaderCorePkg/Stage1A/Ia32/SecEntry.nasm

```diff
@@ -95,14 +95,20 @@ CheckStackRangeDone:
         bts     ebx, 0               ; Set BIT0 in Status
 
 CheckStatusDone:
+        ;
         ; Setup HOB
+        ; This structure has to match the layout of STAGE1A_ASM_PARAM
+        ;
+        push    $0
```

Comment:
can we add comment to state 'higher 32 bit of Status', and the next  for lower 32 bit ?

Reply:
Sounds good -- done!